docs/dev/uap/permissions.rst

```diff
@@ -0,0 +1,90 @@
+Permissions
+===========
+
+Permissions are divided into data access permissions (DAPs) and device permissions. Permissions are checked on a user
+object using the default django.contrib.auth has_perm user-model methods, which take a permission string and an
+optional object for row-level permissions.
```

Comment:
I think I get the point here, but could it somehow be stated differently? Like quoting the original function signature and explaining how it's used?